cmd/sourcepods-dev/dev_setup.go

```diff
@@ -57,6 +63,33 @@ func devSetupAction(c *cli.Context) error {
 
 	return nil
 }
+func genKeys() error {
+	if _, err := os.Stat("./dev/keys/ssh_host_rsa_key"); os.IsNotExist(err) {
+		if err := runCmd("ssh-keygen", "-f", "./dev/keys/ssh_host_rsa_key", "-N", "", "-t", "rsa"); err != nil {
+			return errors.Wrap(err, "can't generate rsa-key")
+		}
+	}
+	if _, err := os.Stat("./dev/keys/ssh_host_dsa_key"); os.IsNotExist(err) {
+		if err := runCmd("ssh-keygen", "-f", "./dev/keys/ssh_host_dsa_key", "-N", "", "-t", "dsa"); err != nil {
+			return errors.Wrap(err, "can't generate dsa-key")
+		}
+	}
+	if _, err := os.Stat("./dev/keys/ssh_host_ecdsa_key"); os.IsNotExist(err) {
+		if err := runCmd("ssh-keygen", "-f", "./dev/keys/ssh_host_ecdsa_key", "-N", "", "-t", "ecdsa", "-b", "521"); err != nil {
+			return errors.Wrap(err, "can't generate ecdsa-key")
+		}
+	}
+	return nil
+}
+
+func runCmd(cmd string, args ...string) error {
```

Comment:
Can we inline this function into the `genKeys()` func above? Let's keep the scope small.